tests/remote/client/test_remote_client.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from remote.client.remote_client import RemoteClient
+
+
+def test_cant_create_instance_of_abc():
+    with pytest.raises(TypeError):
+        RemoteClient()
```

Comment:
Basically unnecessary. Tests that ABC works and doesn't let you instantiate abstract classes.